lib/shifts.rb

```diff
@@ -0,0 +1,39 @@
+require 'date'
+class Shifts
+  attr_reader :key, :date
+
+  def initialize(key = rand(99999), date = Date.today.strftime("%d%m%y"))
+    @key = key
+    @date = date
+  end
```

Comment:
added the `Shifts` class to abstract the functionality of transforming the `key` and `date` into an array of shifts away from the class which uses the shifts to encrypt and decrypt.

Side note, may need to refactor the default values to the `Enigma` class.